src/kem/kyber/ntt.rs

```diff
@@ -17,6 +17,74 @@ const ZETAS_MONTGOMERY_DOMAIN: [KyberFieldElement; 128] = [
     -1530, -1278, 794, -1510, -854, -870, 478, -108, -308, 996, 991, 958, -1460, 1522, 1628,
 ];
 
+// Over time, all invocations of ntt_representation() will be replaced by
+// invocations to this function, upon which this function will be renamed back to
+// ntt_representation().
+#[inline(always)]
+pub(in crate::kem::kyber) fn ntt_with_debug_asserts(
+    mut re: KyberPolynomialRingElement,
+    coefficient_bound: usize,
```

Comment:
Independent whether this should be here or not. It should probably be the same type as `coefficients`.